### PR TITLE
Enable nullable

### PR DESCRIPTION
**Pull request by Pliner**
At the moment, I don't know how to fix models, but anyway it could be done incrementally.

### CODE REVIEW

File: Source/EasyNetQ.Management.Client/EasyNetQ.Management.Client.csproj [hunk 1]
```diff
@@ -49,6 +53,7 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
     <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
+    <PackageReference Include="NullGuard.Fody" Version="3.1.0" PrivateAssets="All" />
```
Comment (zidad) [inline Source/EasyNetQ.Management.Client/EasyNetQ.Management.Client.csproj:56]: Do we want to add a dependency like this to a core package?

**Review by zidad — APPROVED** [review]
Nice!

**Comment by Pliner** [comment]
> Nice!

Thanks 👍🏻

File: Source/EasyNetQ.Management.Client/FodyWeavers.xsd [hunk 2]
```diff
@@ -0,0 +1,68 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (sungam3r) [inline Source/EasyNetQ.Management.Client/FodyWeavers.xsd:1]: You specified `<FodyGenerateXsd>false</FodyGenerateXsd>` but xsd is still here.

**Comment by sungam3r** [comment]
> I don't know how to fix models

`= null!;` is a standard pattenr

**Pull request merged**